ui/admin/app/components/form/auth-method/oidc/index.hbs

```diff
@@ -243,120 +243,105 @@
+                  aria-labelledby='fromClaimHeader'
+                  {{on 'input' (set-from-event claimMap 'from')}}
+                />
+              </B.Td>
+              <B.Td>
                 <Hds::Form::Select::Field
                   @value={{claimMap.to}}
-                  @isInvalid={{@model.errors.account_claim_maps}}
+                  @width='100%'
                   disabled={{form.disabled}}
+                  @isInvalid={{@model.errors.account_claim_maps}}
+                  aria-labelledby='toClaimHeader'
                   {{on 'change' (set-from-event claimMap 'to')}}
                   as |F|
                 >
-                  <F.Label class='sr-only'>{{t 'form.to_claim.label'}}</F.Label>
                   <F.Options>
                     {{#each this.toClaims as |toClaimOption|}}
                       <option
                         value={{toClaimOption}}
                         selected={{eq toClaimOption claimMap.to}}
-                      >{{toClaimOption}}</option>
+                      >
+                        {{toClaimOption}}
+                      </option>
                     {{/each}}
                   </F.Options>
                 </Hds::Form::Select::Field>
-
-              </row.cell>
-              <row.cell @shrink={{true}}>
-                <Rose::Button
-                  @style='warning'
-                  @iconOnly='flight-icons/svg/trash-16'
-                  title={{t 'actions.remove'}}
-                  @disabled={{if
-                    @model.isNew
-                    false
-                    (if form.isEditable false true)
-                  }}
+              </B.Td>
+              <B.Td>
+                <Hds::Button
+                  @text={{t 'actions.remove'}}
+                  @color='critical'
+                  @icon='trash'
+                  @isIconOnly={{true}}
+                  disabled={{form.disabled}}
+                  @isInvalid={{@model.errors.account_claim_maps}}
                   {{on 'click' (fn @removeItemByIndex 'account_claim_maps' i)}}
-                >
-                  {{t 'actions.remove'}}
-                </Rose::Button>
-              </row.cell>
-            </body.row>
+                  data-test-remove-claim-map={{claimMap.from}}
+                />
+              </B.Td>
+            </B.Tr>
           {{/each}}
-
-          <body.row as |row|>
-            <row.cell>
-              <form.input
-                @type='text'
-                @name='from_claim'
+          <B.Tr>
+            <B.Td>
+              <Hds::Form::TextInput::Field
                 @value={{this.newFromClaim}}
-                @label={{t 'resources.auth-method.titles.new-from-claim'}}
+                disabled={{form.disabled}}
+                @isInvalid={{@model.errors.account_claim_maps}}
+                aria-labelledby='fromClaimHeader'
+                {{on 'input' (set-from-event this 'newFromClaim')}}
+                data-test-from-claim-input
```

Comment:
None blocking comment: as the pattern followed in this file, I will suggest to let the `data-test-` selector name just to describe the field, but it does not need the type.
i.e: `data-test-from-clain-input` or `data-test-to-claim-select` can be `data-test-from-claim` and `data-test-to-claim`. So the type of the field is not coupled to its selector.